### PR TITLE
Add OTA update button to Shelly

### PR DESCRIPTION
**Pull request by mib1185**
## Proposed change

This will add the documentation for device services - here the OTA update service.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core/pull/58757
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/shelly.markdown
Comment (thecode) [inline source/_integrations/shelly.markdown]:
```suggestion
- OTA update service does not support battery-powered devices
```

File: source/_integrations/shelly.markdown
Comment (thecode) [inline source/_integrations/shelly.markdown]:
```suggestion
The integration offers device services which can be triggered by a configuration button.
```

File: source/_integrations/shelly.markdown
Comment (thecode) [inline source/_integrations/shelly.markdown]:
```suggestion
Trigger device OTA firmware update.
```
I am not sure if my wording is ok, but I think documentation should not be expressed as "so called"

**Review by thecode — APPROVED** [review]
LGTM once parent PR is merged, Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @mib1185 👍

**Pull request merged**
